ml-prims/test/hmm/magma/b_determinant.cu

```diff
@@ -0,0 +1,127 @@
+// #include <gtest/gtest.h>
```

Comment:
do we need this file if it's fully commented out?

Reply:
You are right, It's not necessary for this PR